utils/interfaces/_library/core.py

```diff
@@ -53,8 +53,6 @@ def __init__(self):
             self.expected_timeout = 5
         elif context.library.library in ("php",):
             self.expected_timeout = 10  # possibly something weird on obfuscator, let increase the delay for now
-        elif context.library.library in ("dotnet",):
```

Comment:
Are you 100% sure about this? It says that 25s after the last HTTP event, the tracer should have sent everything to the agent (instead of 80s).

If you are sure, ok for me !

Reply:
Yes, I'm sure.
We added `DD_INTERNAL_RCM_POLL_INTERVAL` variable to control RCM poll interval, and for tests I set it to 1s.

Reply:
it actually concerns ALL data sent by the tracers : traces, telemetry, etc..